src/components/ActivityCollector/createGetLinkDetails.js

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+export default ({
+  window,
+  getLinkName,
+  getLinkRegion,
+  getAbsoluteUrlFromAnchorElement,
+  findSupportedAnchorElement,
+  determineLinkType
+}) => {
+  return ({ targetElement, config }) => {
+    // Search parent elements for an anchor element
+    // TODO: Replace with generic DOM tool that can fetch configured properties
```

Comment:
What does this TODO mean, and why not use a generic DOM tool now?

Reply:
this does not apply anymore